### DISCUSSION TITLE
post/2023-july-kakaopay-developer-festival/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#31] [Announcements]
# post/2023-july-kakaopay-developer-festival/

ㅋㅍㄱㅍ(카카오페이 개발자 페스티벌) 진행 과정을 소개합니다.

https://tech.kakaopay.com/post/2023-july-kakaopay-developer-festival/

**Comment by yozmngooddev** [comment]
간만에 카카오페이 엔지니어분들과 함께하는 재미있는 자리였어요.
DR 과 개발문화에 관심을 갖고 있는 분들에게도 도움이 됬음 좋겠네요. ^^

**Comment by pay-dan-dy** [comment]
TMI.
퀴즈 예시는 비전과 미션을 바꾸어서 틀린 답이었습니다. 정답은 아래와 같아요~.

카카오페이의 비전은 ‘일상의 모든 금융을 카카오페이 하나로’
미션은 ‘누구에게나 이로운 금융서비스를 만든다.’
